auth-web/src/components/auth/mixins/NextPageMixin.vue

```diff
@@ -50,7 +50,14 @@ export default class NextPageMixin extends Vue {
     switch (this.currentUser?.loginSource) {
       case LoginSource.IDIR:
       case LoginSource.BCROS:
-        return `/${Pages.SEARCH_BUSINESS}`
+        if (this.currentOrganization && this.currentMembership.membershipStatus === MembershipStatus.Active) {
+          if (this.currentMembership.membershipTypeCode === MembershipType.Owner) {
+            return `${Pages.MAIN}/${this.currentOrganization.id}/settings/team-members`
+          } else {
+            return 'director-search-url'
```

Comment:
Is it correct, or should we get the director search web url ?

Reply:
we are redirecting to the url from the TermsOfServiceView page.
this text is for checking the condition, whether we redirect to director search url, or contextually push the route for team management page
please let me know if we should need to change this.